tests/cypress/support/commands_organizations.js

```diff
@@ -0,0 +1,173 @@
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+Cypress.Commands.add('createOrganization', (organizationParams) => {
+    cy.get('.cvat-header-menu-user-dropdown').trigger('mouseover');
+    cy.get('.cvat-header-menu')
+        .should('be.visible')
+        .find('[role="menuitem"]')
+        .filter(':contains("Organization")')
+        .trigger('mouseover');
+    cy.get('.cvat-header-menu-create-organization')
+        .should('be.visible')
+        .click();
+    cy.url().should('contain', '/organizations/create');
+    cy.get('.cvat-create-organization-form').should('be.visible').within(() => {
+        cy.get('#slug').type(organizationParams.shortName);
+        cy.get('#name').type(organizationParams.fullName);
+        cy.get('#description').type(organizationParams.description);
+        cy.get('#email').type(organizationParams.email);
+        cy.get('#phoneNumber').type(organizationParams.phoneNumber);
+        cy.get('#location').type(organizationParams.location);
+        cy.get('[type="submit"]').click();
+    });
+});
+
+Cypress.Commands.add('deletingCreatedOrganizations', (response, otrganizationsToDelete) => {
+    const authKey = response.body.key;
+    cy.request({
+        url: '/api/v1/organizations?page_size=all',
+        headers: {
+            Authorization: `Token ${authKey}`,
+        },
+    }).then((_response) => {
+        const responceResult = _response.body;
+        for (const organization of responceResult) {
+            const organizationId = organization.id;
+            const organizationName = organization.slug;
+            for (const organizationToDelete of otrganizationsToDelete) {
+                if (organizationName === organizationToDelete) {
+                    cy.request({
+                        method: 'DELETE',
+                        url: `/api/v1/organizations/${organizationId}`,
+                        headers: {
+                            Authorization: `Token ${authKey}`,
+                        },
+                    });
+                }
+            }
+        }
+    });
```

Comment:
I have the same comments here like for the functions with users/tasks

Reply:
Done